app/services/submit_application_form.rb

```diff
@@ -33,6 +33,13 @@ def call
       .application_received
       .deliver_later
 
+    if application_form.teaching_authority_provides_written_statement
```

Comment:
```suggestion
    if !application_form.requires_preliminary_check && application_form.teaching_authority_provides_written_statement
```

If I understand the logic correctly, should we only be sending it to users where we don't need a preliminary check?

Reply:
In fact what's the benefit of sending this email given we _don't_ tell this users to not request the document yet:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/32256edb49453e674da3e5bd45abffd1fee29db0/app/views/shared/eligible_region_content_components/_proof_of_recognition.html.erb#L71-L75

Reply:
I think this is a sensible condition to add. I originally thought that the two were mutually exclusive but, for example, Ghanaian applicants would receive this email twice without the additional clause - thanks for spotting this.